packages/catalog-process/src/services/catalogService.ts

```diff
@@ -884,11 +884,14 @@ export async function deleteDraftDescriptorLogic({
     throw notValidDescriptor(descriptorId, descriptor.state.toString());
   }
 
-  const interfacePath = descriptor.docs.find(
-    (doc: Document) => doc.id === descriptorId
-  );
```

Comment:
This logic was wrong, because the `interface` is not stored inside the `docs` array. Instead, the descriptor has a specific `interface` field.